web/src/types/ga-gtag/index.d.ts

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
Unfortunately, `ga-gtag` module does not have any type definitions at [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped), so we are required to write our own declaration file.